cpp/src/strings/filling/fill.cu

```diff
@@ -61,8 +63,8 @@ std::unique_ptr<column> fill( strings_column_view const& strings,
         [d_strings, begin, end, d_value] __device__ (size_type idx) {
             return ((begin <= idx) && (idx < end)) ? !d_value.is_null() : !d_strings.is_null(idx);
         }, stream, mr );
-    rmm::device_buffer null_mask = valid_mask.first;
     auto null_count = valid_mask.second;
+    rmm::device_buffer null_mask(std::move(valid_mask.first));
```

Comment:
is reference variable a better option here instead of move?

Reply:
because another `std::move(null_mask)` is present at line 99.

Reply:
You are right. That would save an extra move. Code has been updated.